postreise/analyze/time.py

```diff
@@ -146,6 +146,7 @@ def change_time_zone(ts, tz):
     except pytz.exceptions.UnknownTimeZoneError:
         raise ValueError("Unknown time zone %s" % tz)
 
+    ts.index.name = tz
```

Comment:
I understand the changes in **postreise/plot/plot_generation_ts_stack.py**, what does this change do? Does this change enable the other ones to work?

Reply:
This change is not essential to make it work. When I was debugging the code, I plot the storage_pg independently without the generation stack on the top and found the name of the index will be shown automatically as the default x-label when we do `df.plot()`, which is always "UTC" no matter what time zone it actually is. I made this change simply for the purpose of removing potential confusion.